types/RestaurantDto.ts

```diff
@@ -2,4 +2,6 @@ export type RestaurantDto = {
     id: number
     name: string
     coordinates: string
+    menu_link: string
+    location: string
```

Comment:
currently restaurants table contains another column location_coordinates, storing the coordinates gotten from the mapbox geocoding api on the first time (but for now can just add the coordinates in by hand)

Reply:
we should consider changing the location_coordinates column, to 2 separate columns for lat and long instead (easier to match the types)